Sources/StreamChatUI/Appearance+ColorPalette.swift

```diff
@@ -33,6 +33,7 @@ public extension Appearance {
         public var background5: UIColor = .streamOverlayDarkStatic
         public var background6: UIColor = .streamGrayWhisper
         public var background7: UIColor = .streamDarkGray
+        public var background8: UIColor = .streamWhite
```

Comment:
> I really don't like these background numbers. They don't provide any context

Agree with you and like the naming neither. AFAIR the approach with numbers was chosen because it was not possible to find a common name for a single color used in several places and step by step this became messy 👎 

Like the idea to have meaningful names but this requires some time spent on investigation and looking for suitable names. On the one hand, I don't see nothing wrong in having a color for concrete purpose (like `messageListBackground`). On there other hand, there should be some `general purpose` for those used across the entire app. The question is what is `general purpose` and what is `component specific`.

Would be nice to dedicate some effort to `ColorPalette` but that is not that urgent since component colors are easily customisable while proper names can be added in minor releases.

*Regarding bubble case: didn't use `popoverBackground` because it was create for popover but is also used in reactions/suggestions/image gallery/file list etc but there was no such `backgroundX`.